src/main/java/reposense/parser/CsvParser.java

```diff
@@ -59,6 +59,11 @@ public CsvParser(Path csvFilePath) throws IOException {
 
             for (int lineNumber = 2; (line = br.readLine()) != null; lineNumber++) {
                 String[] elements = line.split(ELEMENT_SEPARATOR_REGEX);
+                for (int colNum = 0; colNum < elements.length; colNum++) {
+                    if (elements[colNum].startsWith("\"") && elements[colNum].endsWith("\"")) {
```

Comment:
Unfortunately, this detection is also incorrect. What if the origin user input is `"input"`, this `"` is ignored in our case. To solve the problem, you should study how CSV deal with `,`. A proper one should be `element.contains(",")`.  If the split text contains `,`, it is guaranteed that there are start and end `"`.

Reply:
Noted. Made the changes.